nodelet/pkg/pf9kube/pf9/pf9-kube/os.sh

```diff
@@ -89,12 +89,17 @@ fi
  pf9ctr_restart
 }
 
+function set_containerd_sock_permissions_pf9()
+{
+    setfacl -m user:pf9:rwx /run/containerd/containerd.sock
```

Comment:
```suggestion
    setfacl -m user:pf9:rwx /run/containerd/containerd.sock || chmod ...
```
in case `setfacl` command is not present